asciicast/player.go

```diff
@@ -18,14 +21,19 @@ func NewPlayer() Player {
 	return &AsciicastPlayer{Terminal: terminal.NewTerminal()}
 }
 
-func (r *AsciicastPlayer) Play(path string) error {
+func (r *AsciicastPlayer) Play(path string, maxWait uint) error {
 	asciicast, err := Load(path)
 	if err != nil {
 		return err
 	}
 
+	adjustedMaxWait := float64(maxWait)
+	if adjustedMaxWait <= 0 {
+		adjustedMaxWait = playbackDefaultMaxWait
+	}
+
 	for _, frame := range asciicast.Stdout {
-		delay := time.Duration(float64(time.Second) * frame.Delay)
+		delay := time.Duration(float64(time.Second) * math.Min(adjustedMaxWait, frame.Delay))
```

Comment:
I'd rather not modify delay at all when maxWait is 0. Here you modify it when it's > 3600. 

What about something like this:

```go
delay := frame.Delay
if maxWait > 0 && delay > maxWait {
  delay = maxWait
}
time.Sleep(time.Duration(float64(time.Second) * delay))
```

It's similar to what happens during recording: https://github.com/asciinema/asciinema/blob/master/asciicast/stream.go#L41-L45